layouts/community/ergodox/bocaj/config.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
-
 #include QMK_KEYBOARD_CONFIG_H
 
 #define LEADER_TIMEOUT 250
 #define LEADER_PER_KEY_TIMING
+#undef TAP_DANCE_ENABLE
```

Comment:
This isn't were to set this.

You'd want to use the rules.mk to disable tapdance.

Reply:
Thank you, it had already been set in rules.mk. Recent commit has removed this line